go.mod

```diff
@@ -10,7 +10,7 @@ go 1.16
 // -----------------------------------------------------------------------------
 
 // client-go does not autoresolve versions correctly due to a historical versioning problem upstream
-replace k8s.io/client-go => k8s.io/client-go v0.20.3
+replace k8s.io/client-go => k8s.io/client-go v0.20.5
```

Comment:
Just realized that, when reviewing, I missed that 50a5bb9 re-added the `replace` directive that af95eab removed right before :facepalm: .

With Go 1.16, we should be able to remove this `replace` once and for all. This requires bumping Go to 1.16 in KTF and removing the replace directive there. Let's talk about this in the upcoming zoom.

Reply:
```suggestion
```